PopupBridge/src/main/java/com/braintreepayments/api/PopupBridgeClient.kt

```diff
@@ -6,25 +6,45 @@ import android.net.Uri
 import android.webkit.WebView
 import androidx.core.net.toUri
 import androidx.fragment.app.FragmentActivity
+import androidx.lifecycle.lifecycleScope
+import com.braintreepayments.api.internal.PendingRequestRepository
 import com.braintreepayments.api.internal.PopupBridgeJavascriptInterface
 import com.braintreepayments.api.internal.PopupBridgeJavascriptInterface.Companion.POPUP_BRIDGE_URL_HOST
 import com.braintreepayments.api.internal.isVenmoInstalled
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 import org.json.JSONException
 import org.json.JSONObject
 import java.lang.ref.WeakReference
 
 class PopupBridgeClient @SuppressLint("SetJavaScriptEnabled") internal constructor(
-    private val activityRef: WeakReference<FragmentActivity>,
-    private val webViewRef: WeakReference<WebView>,
+    activity: FragmentActivity,
+    webView: WebView,
```

Comment:
I moved the WeakReferences out of the constructor to make it easier to reference `activity` in the default values of the constructor. WeakReference can return a null value which needed to be handled when referencing the activity.